wallets/rn_cli_wallet/src/utils/wagmigotchiABI.ts

```diff
@@ -0,0 +1,94 @@
+export default [
+  {inputs: [], stateMutability: 'nonpayable', type: 'constructor'},
```

Comment:
are you using this file?

Reply:
Wagmi with the `usdtAbi.ts` file, these might be auto generated (not really sure) since I don't remember I moved these to here or manually created. Let me double check if they are necessary

Reply:
this is a file im using in the W3MWagmi example to test contract interaction, so its probably not needed